www/api/index.php

```diff
@@ -88,7 +88,7 @@
     }
 
     /* Insert Token in API webservice session table */
-    $token = base64_encode(uniqid('', true));
+    $token = base64_encode(random_bytes(32));
```

Comment:
random_bytes isn't supported in php 5.4

Reply:
you are completely right here! going to apply a fix for it, thanks